pkg/apis/config/defaults.go

```diff
@@ -44,6 +44,13 @@ const (
 	// DefaultUserContainerName is the default name we give to the container
 	// specified by the user, if `name:` is omitted.
 	DefaultUserContainerName = "user-container"
+
+	// DefaultContainerConcurrency is the default container concurrency. It will be set if ContainerConcurrency is not specified.
+	DefaultContainerConcurrency int64 = 0
+
+	// DefaultMaxRevisionContainerConcurrency is the maximum configurable
+	// container concurrency.
+	DefaultMaxRevisionContainerConcurrency int64 = 1000
```

Comment:
This actually can probably be removed altogether, since we switched to infinity breaker.

Reply:
@vagababov are you suggesting removing `DefaultMaxRevisionContainerConcurrency`, and don't check for upper bound in `ValidateContainerConcurrency(...)`?

Reply:
NM, I'll fix it in https://github.com/knative/serving/issues/5161